peachjam/templates/peachjam/_contact_form.html

```diff
@@ -0,0 +1,46 @@
+{% load i18n %}
+<section class="row ">
+  <div class="my-4 col-lg-6 card">
```

Comment:
I suggest dropping the card; if you do want the card, don't put it on the col.

Also add: `col-12`